internal/provider/route_test.go

```diff
@@ -110,6 +113,10 @@ func TestConvertRoute(t *testing.T) {
 			TLSCustomCAKeyPairID:                      types.StringValue("custom-ca-1"),
 			KubernetesServiceAccountTokenFile:         types.StringValue("/path/to/token"),
 			JWTIssuerFormat:                           types.StringValue("IssuerURI"),
+			BearerTokenFormat:                         types.StringValue("idp_access_token"),
```

Comment:
should also add this field to the subtest below to test converting in the other direction